introspection/nm-manager.xml

```diff
@@ -381,6 +381,19 @@
     <property name="Version" type="s" access="read"/>
 
     <!--
+        Capabilities:
+
+        The current set of capabilities.  This is a list of 32 bit
+        integers where specific bits are set when a capability is
+        currently available.  For example, capability number 42 would
+        be represented by bit (42 % 32) == 10 in the element with index
+        (42 / 32) == 1 in the array.
+
+        See <link linkend="NMCapability">NMCapability</link> for currently defined capability numbers.
+    -->
+    <property name="Capabilities" type="au" access="read"/>
```

Comment:
this is certainly a way to do it... I like that we can pack 32 capabilities in one number and it's reasonably simple to access the API from the client-side.

On the other hand, I was wondering if we want to reserve a range of capabilities for downstream users. In that case, those capabilities would have a high number, and could not be easily expressed in a bitfield. That is the reason why I was thinking about exposing every capability as individual number. Hm, probably the bitfield *is* nicer to use and more efficient. What are other opinions here?